### ISSUE TITLE
The fine-tune accuracy is a little bit lower 

### ISSUE THREAD

**Issue by Wallace-222** [facebookresearch/mae#105]
I have a similar question with #91  and #30 , the result of my own runs is a little bit lower than the reported one here (about 0.2%~0.5%). For example, for vit_large, I only obtain 85.2% for 800-epoch pre-training, compared with the 85.4% in the paper and the 85.5% shown here. Do the authors have any hints about this mismatch?
